iwyu_preprocessor.cc

```diff
@@ -285,6 +285,13 @@ void IwyuPreprocessorInfo::HandlePragmaComment(SourceRange comment_range) {
     return;
   }
 
+  if (MatchOneToken(tokens, "associated", 1, begin_loc)) {
```

Comment:
We use Google style, with two-space indentation. We have a clang-format configuration file, so you can get clang-format and run it over your specific changes (please don't run it on the whole file, because it will probably generate lots of noise in the diff). git-clang-format can be helpful here, I think, if you can get it to work.